src/ConstructStyleSheet.js

```diff
@@ -135,7 +119,43 @@ export default class ConstructStyleSheet {
   }
 }
 
+// Implement all methods from the base CSSStyleSheet constructor as
+// a proxy to the raw style element created during construction.
+cssStyleSheetMethods.forEach(method => {
+  ConstructStyleSheet.prototype[method] = function() {
+    if (!sheetMetadataRegistry.has(this)) {
+      throw new Error(
+        `Failed to execute '${method}' on 'CSSStyleSheet': Can't call ${method} on non-constructed CSSStyleSheets.`,
+      )
+    }
+
+    const args = arguments;
+    const { adopters, actions, basicStyleElement } = sheetMetadataRegistry.get(this);
+    const result = basicStyleElement.sheet[method].apply(basicStyleElement.sheet, args);
+
+    adopters.forEach(styleElement => {
+      if (styleElement.sheet) {
+        styleElement.sheet[method].apply(styleElement.sheet, args);
+      }
+    });
+
+    actions.push([method, args]);
+
+    return result;
+  }
+});
+
+export function instanceOfStyleSheet(instance) {
+  return (
+    instance.constructor === ConstructStyleSheet ||
+    instance instanceof OldCSSStyleSheet ||
+    (frame.CSSStyleSheet && instance instanceof frame.CSSStyleSheet)
+  );
+}
```

Comment:
In reference to the discussion here: https://github.com/calebdwilliams/construct-style-sheets/pull/46#discussion_r377300040 - I think the use and implementation of this method is no longer relevant. We are effectively doing the following:

```js
class ConstructStylesSheet {
    replace() { ... }
    replaceSynce() { ... }
    addRule() { ... }
    removeRule() { ... }
    ... et al
}

window.CSSStyleSheet = ConstructStyleSheet;
```
So all checks can simply do `instanceof ConstructStyleSheet` - no need for for this method or the `Symbol.hasInstance` stuff.

Reply:
Sorry I've been MIA on this, but I believe the impetus for the instanceof stuff is because we still want 

```javascript
const style = document.createElement('style');
document.body.append(style);

style.sheet instanceof CSSStyleSheet;
```

and

```javascript
const sheet = new CSSStyleSheet();
sheet instanceof CSSStyleSheet
```

to both be true. Can you verify your code does that?